apis/bases/core.openstack.org_openstackcontrolplanes.yaml

```diff
@@ -564,6 +564,9 @@ spec:
                   enabled:
                     default: true
                     type: boolean
+                  randomPodNames:
```

Comment:
this looks like getting away from standard kubernetes resources. Statefulsets guarantee stable names, deployments, dsets, and replicasets are about to use dynamic names for pods.
Please do not introduce artifical CRD parameters, maybe instead allow choosing the kind of the resource it need to create, like deployment, ds, or sts?

Reply:
These are stable names, the `random` part is not really true.
The value is based on the 5 first characters of the UID of the `OpenStackControlPlane` CR.
It doesn't change even if everything is manually destroyed and the openstack-operator recreates it.
The pods and CRs will have a stable name.

I'm open to changing the name if you feel like it's not clear, but to be honest I don't know of a better name.

Reply:
The name should be connected to the kind of the managed resource, for pods, either sts, or ds, or deployment.
We shouldn't invent custom resouces controllers with custom nameing schemes

Reply:
How about a `boundName`? As it's bound to the control plane?

Reply:
All those names sound theoretically nice, but I doubt customers will understand what the field is meant to do, because even I have trouble figuring it out just from the name.

Since the `random` part of the name sounds weird, we could change it to `uniquePodNames`.

Reply:
I agree that the primary goal is to have customers understand what the boolean does. We could change to `uniquePodNames` if it sounds better (I'm bad at that kind of things :/) but I think the patch goes to the right direction.
Only `Glance` and `Cinder` sections have the parameter, which is set to `false` (and the job executions prove that there are no regressions associated to this code).